.github/actions/deploy-environment/action.yml

```diff
@@ -56,8 +56,10 @@ runs:
       if: ${{ inputs.pull-request-number != '' }}
       shell: bash
       run: |
-        az aks get-credentials -g s189t01-tsc-ts-rg -n s189t01-tsc-test-aks
+        make ci review get-cluster-credentials
         kubectl exec -n tra-development deployment/apply-for-qts-review-${{ inputs.pull-request-number }}-web -- sh -c "cd /app && /usr/local/bin/bundle exec rails db:seed review_app:configure example_data:generate"
```

Comment:
`${{ inputs.pull-request-number }}` can be replaced by `${PULL_REQUEST_NUMBER}`

Reply:
updated